src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -6,6 +6,14 @@
         <Section class="'pt-0'" title.translate="Values">
           <ChartShowValues t-props="props"/>
         </Section>
+        <Section class="'pt-1'" title.translate="Number formatting">
+          <Checkbox
+            name="'humanizeNumbers'"
+            label.translate="Humanize numbers"
```

Comment:
Not sure the label is very clear for a user point of view

Reply:
I agree, but it's the same as the already used label in the scorecard that does the same things, so I chose to use it for consistency :/

Reply:
Ok, good for me then